app/helpers/users_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module UsersHelper
+  def my_posts
+    current_user.posts.all
+  end
+end
```

Comment:
- [ ] There should be NO application logic in the views nor helpers.

You should move this to the user model as an instance method. Then in your controller, you initialize an instance variable to be used in your view.